### DISCUSSION TITLE
Any full blown modules available?

### DISCUSSION THREAD

**Discussion by abukres** [oqtane/oqtane.framework#1852] [General]
I see there was a post by Shaun about the module marketplace. Does it exist? I didn't see it on the site nor in the app.
I would like to see if there are any production modules out there. I think there's 2sxc.

**Comment by hishamco** [comment]
Oqtane has two open source modules:

1. https://github.com/oqtane/oqtane.blogs
2. https://github.com/oqtane/Oqtane.Survey

Also @leigh-pointer showed StudioElf modules but I'm not sure if some are open source or not

**Reply by abukres** [reply]
> Oqtane has two open source modules:
> 
> 1. https://github.com/oqtane/oqtane.blogs
> 2. https://github.com/oqtane/Oqtane.Survey
> 
> Also @leigh-pointer showed StudioElf modules but I'm not sure if some are open source or not

Thanks. I meant more like commercial type modules. A module you would pay for.

**Reply by hishamco** [reply]
I remember already started sort of Oqtane gallery for free and paid modules, I'm not sure if there's any available yet

**Comment by sbwalker** [comment]
@abukres Oqtane has taken a modern approach to third party extensions - it has an integrated "app store" much like modern mobile phone or game platforms. A host user can navigate to Module Management or Theme Management within the application and choose to install a new extension. There are already a number of free third party extensions available for immediate download and a commercial marketplace is currently in beta:

![image](https://user-images.githubusercontent.com/4840590/144878281-a9a53f04-45d6-40b2-8a89-fecff8d7bbc2.png)

You can also view a list of Oqtane extensions on https://www.oqtane.net ( however they can only be installed through the framework itself )

**Reply by leigh-pointer** [reply]
I will be releasing soon a  production modules one is a Project Backlog application

![AnimationMyBacklog](https://user-images.githubusercontent.com/9447612/144904006-3e5ee69d-9302-4678-8335-88ee5ca9b8e1.gif)

 the other is a Dynamic Forms and Data appliction.

![AnimationDynamicForms](https://user-images.githubusercontent.com/9447612/144908784-5461925c-a455-402e-8047-730b13d54db2.gif)

**Reply by rchibnik** [reply]
Looking forward to trying these
